// lodash-babel/forOwnRight.js
"use strict";

exports.__esModule = true;
exports.default = void 0;

var _baseForOwnRight = _interopRequireDefault(require("./_baseForOwnRight.js"));

var _castFunction = _interopRequireDefault(require("./_castFunction.js"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

/**
 * This method is like `_.forOwn` except that it iterates over properties of
 * `object` in the opposite order.
 *
 * @static
 * @memberOf _
 * @since 2.0.0
 * @category Object
 * @param {Object} object The object to iterate over.
 * @param {Function} [iteratee=_.identity] The function invoked per iteration.
 * @returns {Object} Returns `object`.
 * @see _.forOwn
 * @example
 *
 * function Foo() {
 *   this.a = 1;
 *   this.b = 2;
 * }
 *
 * Foo.prototype.c = 3;
 *
 * _.forOwnRight(new Foo, function(value, key) {
 *   console.log(key);
 * });
 * // => Logs 'b' then 'a' assuming `_.forOwn` logs 'a' then 'b'.
 */
function forOwnRight(object, iteratee) {
  return object && (0, _baseForOwnRight.default)(object, (0, _castFunction.default)(iteratee));
}

var _default = forOwnRight;
exports.default = _default;
